ts/Extensions/DashboardPlugins/HighchartsComponent.ts

```diff
@@ -602,25 +608,46 @@ namespace HighchartsComponent {
     }>;
 
     export interface ComponentOptions extends Component.ComponentOptions, EditableOptions {
+        /**
+         * Declares update of the store, when edit chart.
```

Comment:
Please check the description where the default value is set, update this and remove the one on the top.